src/Mailgun/Model/Tag/Tag.php

```diff
@@ -37,7 +37,7 @@ class Tag
      * @param \DateTime $firstSeen
      * @param \DateTime $lastSeen
      */
-    public function __construct($tag, $description, \DateTime $firstSeen, \DateTime $lastSeen)
+    public function __construct($tag, $description, \DateTime $firstSeen = null, \DateTime $lastSeen = null)
```

Comment:
Hey @Nyholm 

I can't see any PHP Unit test to create new tags without the dates.

Are we sure this case might happen?